compose/cli/docker_client.py

```diff
@@ -32,4 +32,4 @@ def docker_client():
         )
 
     timeout = int(os.environ.get('DOCKER_CLIENT_TIMEOUT', 60))
-    return Client(base_url=base_url, tls=tls_config, version='1.14', timeout=timeout)
+    return Client(base_url=base_url, tls=tls_config, version='1.17', timeout=timeout)
```

Comment:
That is a pretty serious sadface :(  Is there a published schedule for that transition or an unpublished estimated date?

Thanks
-steve